src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Stream.of(s.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
```

Comment:
For the future, do not use abbreviations, fix everywhere
```suggestion
                .flatMap(lineNumbers -> Stream.of(lineNumbers.split(",")))
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```